build.cmd

```diff
@@ -227,19 +229,18 @@ if '%RestorePackages%' == 'true' (
 )
 
 
+rem Always do this it installs the dotnet cli and publishes the LKG
+rem ===============================================================
+
 set DOTNET_HOME=.\packages\dotnet
 set _dotnetexe=.\packages\dotnet\dotnet.exe
-if '%BUILD_CORECLR%' == '1' (
-  rem check to see if the dotnet cli tool exists
-  if not exist %_dotnetexe% (
-    echo Error: Could not find %_dotnetexe%.
-    rem do zipfile install nonsense
-    if not exist packages ( md packages )
-    if exist packages\dotnet ( rd packages /s /q )
-    powershell.exe -executionpolicy unrestricted -command .\scripts\install-dotnetcli.ps1 https://dotnetcli.blob.core.windows.net/dotnet/beta/Binaries/Latest/dotnet-dev-win-x64.latest.zip packages
-    @if ERRORLEVEL 1 echo Error: fetch dotnetcli failed && goto :failure
-    popd
-  )
+rem check to see if the dotnet cli tool exists
+if not exist %_dotnetexe% (
+  echo Could not find %_dotnetexe%. Do zipfile install
+  if not exist packages ( md packages )
+  if exist packages\dotnet ( rd packages\dotnet /s /q )
+  powershell.exe -executionpolicy unrestricted -command .\scripts\install-dotnetcli.ps1 https://dotnetcli.blob.core.windows.net/dotnet/beta/Binaries/Latest/dotnet-dev-win-x64.latest.zip packages
+  @if ERRORLEVEL 1 echo Error: fetch dotnetcli failed && goto :failure
```

Comment:
This requires an download + installation of CoreCLR for all users of the repo, even when not doing anything to do with CoreCLR, e.g. when trying to fix bugs.  Since CoreCLR is about 5GB (or seems to be, I stopped the download after 20 minutes after nearing 1GB) I don't think we can require it as a download in all development paths.